src/main/java/com/uid2/shared/auth/OperatorKey.java

```diff
@@ -43,8 +56,13 @@ public OperatorKey(String key, String name, String contact, String protocol, lon
     public long getCreated() { return created; }
     public boolean isDisabled() { return disabled; }
     public void setDisabled(boolean disabled) { this.disabled = disabled; }
+    public Set<Role> getRoles() {
+        return roles;
+    }
+
     public Integer getSiteId() { return siteId; }
     public void setSiteId(Integer siteId) { this.siteId = siteId; }
+    public OperatorKey withRoles(Set<Role> roles) { this.roles = Collections.unmodifiableSet(roles); return this; }
```

Comment:
Might be better to just add a setter like the other fields:
`public void setRoles(Set<Role> roles) { ... }`